src/pages/questionPage.js

```diff
@@ -1,72 +1,79 @@
-'use strict';
+"use strict";
 
-import { ANSWERS_LIST_ID } from '../constants.js';
-import { NEXT_QUESTION_BUTTON_ID } from '../constants.js';
-import { getQuestionElement } from '../views/questionView.js';
-import { createAnswerElement } from '../views/answerView.js';
-import { createCheatButton, showCorrectAnswerElement } from '../views/cheatAnswerView.js';
-import { quizData } from '../data.js';
-import { router } from '../router.js';
+import { ANSWERS_LIST_ID } from "../constants.js";
+import { NEXT_QUESTION_BUTTON_ID } from "../constants.js";
+import { getQuestionElement } from "../views/questionView.js";
+import { createAnswerElement } from "../views/answerView.js";
+import {
+  createCheatButton,
+  showCorrectAnswerElement,
+} from "../views/cheatAnswerView.js";
+import { quizData } from "../data.js";
+import { router } from "../router.js";
 
 export const initQuestionPage = (userInterface) => {
-const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
-const questionElement = getQuestionElement(currentQuestion.text);
-userInterface.appendChild(questionElement);
-const answersListElement = document.getElementById(ANSWERS_LIST_ID);
-for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
-  const answerElement = createAnswerElement(key, answerText);
-  answersListElement.appendChild(answerElement);
-  answerElement.addEventListener('click',getAnswer)
-}
-  
-// get next question handler
-document
+  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+  const questionElement = getQuestionElement(currentQuestion.text);
+  userInterface.appendChild(questionElement);
+  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+  for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
+    const answerElement = createAnswerElement(key, answerText);
+    if (key === currentQuestion.correct) {
+      answerElement.setAttribute("data-correct", "correct");
+    }
+    answersListElement.appendChild(answerElement);
+    answerElement.addEventListener("click", getAnswer);
+  }
+
+  // get next question handler
+  document
     .getElementById(NEXT_QUESTION_BUTTON_ID)
-    .addEventListener('click', isAnswerSelected);
+    .addEventListener("click", isAnswerSelected);
 };
-
-// check answers if correct or not 
+// check answers if correct or not
 let isSelectedOneAnswer = false;
```

Comment:
This is what I wrote yesterday in the chat. In order to check if user selected an answer when clicked ‘next’ , you created this new variable `let isSelectedOneAnswer = false;` . You don’t actually need it, since in data.js each question has a selected key that is null unless the user has selected an answer. Can you use that?